readme.md

```diff
@@ -885,6 +885,10 @@ The list of available parse options:
 
 By default parser skips the whitespace characters between the terms, this can be changed using this option.
 
+* set_dont_skip_newline(bool value)
+
+If set_skip_whitespace is set to ```true```, this options allows to not skip newline (```\n```) characters.
```

Comment:
Change to 'skip_newline', defaulting to true.
Add notice about option setting ignored if skip_whitespace is false. (preferably as a note markup syntax to bring more attention, there are several used in the readme file)